cscs-checks/microbenchmarks/osu/src/Makefile_p2p

```diff
@@ -0,0 +1,25 @@
+EXECUTABLES := p2p_osu_bw p2p_osu_latency
+
+all: $(EXECUTABLES)
+
+SRCS += osu_util.c \
+        osu_latency.c \
+        osu_bw.c
+
+OBJS_BW = osu_util.o osu_bw.o
+OBJS_LT = osu_util.o osu_latency.o
+
+OBJS := $(SRCS:.c=.o)
+
+$(OBJS):
+	$(CC) $(CPPFLAGS) -o $(@) -c $(@:.o=.c)
+
+p2p_osu_bw: $(OBJS_BW)
+	$(CC) $(CPPFLAGS) $(LDFLAGS) $(OBJS_BW) -o $(@)
```

Comment:
You should still pass the `CXXFLAGS` if you want to be completely correct. `CPPFLAGS` are the *preprocessor* flags, i.e., the `-D` and `-I` options mostly.